SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -7,7 +7,7 @@ Name:           pyproject-rpm-macros
 #   Increment Z when this is a bugfix or a cosmetic change
 # Dropping support for EOL Fedoras is *not* considered a breaking change
 Version:        1.0.0~rc1
-Release:        3%{?dist}
+Release:        4%{?dist}
```

Comment:
The toolchain-manfiests need a bump for `pyproject-rpm-macros`

Reply:
Manifest files updated.